### PR TITLE
ci: static checks: use defined target_branch

### PR DESCRIPTION
**Pull request by snir911**
Fixes: #2214
Signed-off-by: Snir Sheriber <ssheribe@redhat.com>

### CODE REVIEW

**Review by fidencio — APPROVED** [review]
lgtm, thanks @snir911!

File: .github/workflows/static-checks.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
       TRAVIS_PULL_REQUEST_BRANCH: ${{ github.head_ref }}
       TRAVIS_PULL_REQUEST_SHA : ${{ github.event.pull_request.head.sha }}
       RUST_BACKTRACE: "1"
-      target_branch: ${TRAVIS_BRANCH}
```
Comment (wainersm) [inline .github/workflows/static-checks.yaml:16]: `TRAVIS_BRANCH` is set at line 12 to `github.base_ref`. At this point its value is not available?

Comment (snir911) [reply .github/workflows/static-checks.yaml:16]: No, I changed it after it seemed to fail to evaluate it as environment variable, maybe it's possible with some special syntax but i couldn't find something like that.

**Review by jodh-intel — APPROVED** [review]
Thanks @snir911.

lgtm

**Pull request merged**